bundle/jsky.app.ot/src/main/java/jsky/app/ot/editor/seq/DynamicSequenceTableModel.java

```diff
@@ -179,18 +175,18 @@ public int getColumnCount() {
         return _iteratedKeys.length;
     }
 
-    private boolean isGcal(Config config) {
-        String type = (String) config.getItemValue(OBS_TYPE_KEY);
+    private boolean isGcal(final Config config) {
+        final String type = (String) config.getItemValue(OBS_TYPE_KEY);
         return InstConstants.FLAT_OBSERVE_TYPE.equals(type) ||
                InstConstants.ARC_OBSERVE_TYPE.equals(type);
     }
 
-    public Object getValueAt(int rowIndex, int columnIndex) {
+    public Object getValueAt(final int rowIndex, final int columnIndex) {
         if (_sequence == null) return null;
         if (_sequence.size() == 0) return null;
 
-        Config config = _sequence.getStep(rowIndex);
-        ItemKey   key = _iteratedKeys[columnIndex];
+        final Config config = _sequence.getStep(rowIndex);
+        final ItemKey   key = _iteratedKeys[columnIndex];
         if (!isGcal(config) && "calibration".equals(key.getParent().toString())) return "";
```

Comment:
Yeah, I saw that warning. Didn't really know what to do about it, because it doesn't seem to be an actual problem. So I silently ignored it.. ;) I can give it another look.